torchvision/ops/deform_conv.py

```diff
@@ -0,0 +1,70 @@
+from typing import Tuple
+
+import torch
+from torch import nn, Tensor
+from torch.nn.modules.utils import _pair
+from torch.jit.annotations import List
+
+
+def deform_conv(input, offset, weight, stride=(1, 1), pad=(0, 0), dilation=(1, 1), n_parallel_imgs=64):
+    # type: (Tensor, Tensor, Tensor, Tuple[int, int], Tuple[int, int], Tuple[int, int], int) -> Tensor
+    """
+    Performs Deformable Convolution, described in Deformable Convolutional Networks
+
+    Arguments:
+        input (Tensor[batch_sz, in_channels, in_h, in_w]): input tensor
+        offset (Tensor[batch_sz, 2 * n_offset_grps * weight_h * weight_w, out_h, out_w])
+        weight (Tensor[out_channels, in_channels // n_weight_grps, weight_h, weight_w]):
+            convolution weights, with n_weight_grps different connection groups
+        stride (int or Tuple[int, int]): distance between convolution centers
+        pad (int or Tuple[int, int]): height/width of padding of zeroes around each image
+        dilation (int or Tuple[int, int]): point distance in convolution grid
+        n_parallel_imgs (int): Number of images to be processed at once; does not change
```

Comment:
I'm not sure this is something that should be exposed to the user.
In PyTorch, we try to limit the number of knobs that we expose to the users and we try to use good defaults instead.

While I understand the need for this (if we just make by default `n_parallel_imgs = batch_size` the amount of memory used will be pretty large), I think we should either let it 1 or `batch_size` by default, or just make a reasonable guess.

The reason why I'm saying this is because in PyTorch, `nn.functional.conv2d` doesn't expose such options, and I think we should follow accordingly.